<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLIENTClass</class>
 <widget class="QMainWindow" name="CLIENTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CLIENT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_move">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse Move</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_press">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse Press</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_release">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse Release</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_double_click">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse Double Click</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_c1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>connect 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_c2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>connect 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_c3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>connect 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="key_press">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Key press:</string>
    </property>
   </widget>
   <widget class="QLabel" name="key_release">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Key release</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CLIENT.qrc"/>
 </resources>
 <connections/>
</ui>
